appveyor.yml

```diff
@@ -35,7 +35,7 @@ before_build:
 build_script:
     - cmd: |
         set GRADLE_OPTS="-Dorg.gradle.jvmargs='-XX:MaxMetaspaceSize=512m'"
-        gradlew.bat clean install check -Ptest.exclude="**/*15*" -Druntime=%RUNTIME% -DruntimeVersion=%RUNTIME_VERSION% --stacktrace --info --no-daemon
+        gradlew.bat clean install check -Ptest.exclude="**/*15*,**/Polling*,**/*estLoose*" -Druntime=%RUNTIME% -DruntimeVersion=%RUNTIME_VERSION% --stacktrace --info --no-daemon
         gradlew.bat wrapper --gradle-version 4.10
         gradlew.bat check -Ptest.include="**/*15*" -Druntime=%RUNTIME% -DruntimeVersion=%RUNTIME_VERSION% --stacktrace --info --no-daemon
```

Comment:
One more thought...can you see if any of the tests you are excluding actually work with this older gradle wrapper?

Reply:
Seems like the TestLoose* still fails when using gradle 4.10, and the PollingDevTest ran fine until it errored on cleanup.

Will see if any of the other jobs or reruns complete with a success on PollingDevTest.